src/interfaces/wallet.h

```diff
@@ -408,6 +408,7 @@ struct WalletTx
     int64_t time;
     std::map<std::string, std::string> value_map;
     bool is_coinbase;
+    bool is_platform_transfer;
```

Comment:
I know none of the others do it, but we should ensure these are always initialized
```suggestion
    bool is_platform_transfer{false};
```

Reply:
please squash 9748a048b41a83be9703bf76d69ca46f73330d31 into a9a7f0766844543c90e249896f5ca10243c56c1d

Reply:
done in 9748a048b41...21f174aff10